### PR TITLE
Sound Enhancement Add ETL/VRS Sound

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgSounds.hpp [hunk 1]
```diff
@@ -136,4 +136,18 @@ class CfgSounds
...
+	{
+	    sound[] = {"\fza_ah64_us\audio\CreakingAirFrame.ogg", 1, 1};
+	    frequency = "CustomSoundController3";
+	    volume = "(1-camPos)*rotorSpeed*CustomSoundController4";
+	};
+	class ETL_VRS_Shake_02
+	{
+	    sound[] = {"A3\Sounds_F\vehicles\noises\vehicle_stress3", 1, 1};
+	    frequency = "CustomSoundController3";
+	    volume = "(1-camPos)*rotorSpeed*CustomSoundController4";
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgSounds.hpp:150]: Do these work as set here? And how are they set to only work for the Apache?

File: @AH-64D Apache Longbow/Addons/fza_ah64_us/audio/CreakingAirFrame.ogg
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_us/audio/CreakingAirFrame.ogg:1]: Where did we get this from?

File: @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgSounds.hpp (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_controls/config/cfgSounds.hpp:150]: Moved to their correct location

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf [hunk 2]
```diff
@@ -112,6 +112,11 @@ if (_velXY > 8.23 && _velXY < 12.35) then {
     setCamShakeParams [0.0, 0.5, 0.0, 0.0, true];
     addCamShake       [2.5, 1, 5];
     enableCamShake false;
+
+    SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:116]: You set these custom controller to always be 6.4, could this instead be set in config?

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf [hunk 3]
```diff
@@ -122,29 +127,43 @@ if (_velXY < 12.35) then {  //must be less than ETL
         setCamShakeParams [0.0, 0.5, 0.0, 0.0, true];
         addCamShake       [2.5, 1, 5];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
+        SetCustomSoundController[_heli, "CustomSoundController4", 1.8];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:132]: I could see us using custom sound controllers a lot in the future.

Let's take a look and see if we can put each number / name in a preprocessor macro and share it, similar to MFD indexes, to make it easier to read

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf [hunk 4]
```diff
@@ -122,29 +127,43 @@ if (_velXY < 12.35) then {  //must be less than ETL
...
     };
     //> 4800fpm
     if (_velZ < -24.384) then {
         enableCamShake true;
         setCamShakeParams [0.0, 1.0, 0.0, 2.0, true];
         addCamShake       [4.0, 1, 6.5];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:161]: Why have we started at 3, are there 2 more I can't see in the diff or are they reserved?

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf [hunk 5]
```diff
@@ -122,29 +127,43 @@ if (_velXY < 12.35) then {  //must be less than ETL
...
+        SetCustomSoundController[_heli, "CustomSoundController4", 1.8];
     };
     //3867fpm to 4800 fpm
     if (_velZ <= -19.64 && _velZ > -24.384) then {
         enableCamShake true;
         setCamShakeParams [0.0, 0.75, 0.0, 0.75, true];
         addCamShake       [3.5, 1, 6.0];
         enableCamShake false;
+
+        SetCustomSoundController[_heli, "CustomSoundController3", 6.4];
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:151]: Generally we don't capitalise the start of commands - e.g. `setCustomSoundController`

**Review by mattysmith22 — CHANGES_REQUESTED** [review]
Looks good so far!

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf (hunk 2 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:116]: Will move once tuning is completed

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf (hunk 3 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:132]: Sounds good to me

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf (hunk 5 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:151]: Fixed

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf (hunk 4 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf:161]: I copied Arens code and didn't want to break it....

**Review by mattysmith22 — APPROVED** [review]

**Pull request merged**